.github/workflows/build-images.yml

```diff
@@ -257,7 +257,7 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: minimal_vertica_image
       with:
-        default: docker.io/opentext/vertica-k8s-private:latest-minimal-master
+        default: docker.io/opentext/vertica-k8s-private:20240606-minimal
```

Comment:
It's better if we change only the `vertica-image` under `with` clause of `e2e-leg-9-vcluster`, as that's the only test we want to fix? Changing this line in `build-server-minimal` would potentially impact all places where we use server minimal images.

Reply:
I already did that, but we will have trouble with authorization for Docker login to pull the image. We need to sign in to Docker with a username and password to pull the image in build-images.yaml

Reply:
As we discussed, adding `docker.io` as a prefix of the image name solves the problem.

Reply:
it seems it works with docker.io :)